samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/auth.js

```diff
@@ -92,3 +92,28 @@ async function getTokenSilently() {
         });
     }
 }
+
+async function loadTokens() {
```

Comment:
I would not put this in the sample itself because in a real world use case you wouldn't want to have your actual app use this API. Instead the test file should call the ROPC flow and load the tokens into the cache using the puppeteer APIs

Reply:
After working through e2e tests with Thomas, we have decided to scope this feature to be browser environments only. E2E tests have been removed.